src/System.Net.HttpListener/tests/System.Net.HttpListener.Tests.csproj

```diff
@@ -10,26 +10,26 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='netstandard-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="GetContextHelper.cs" />
+    <Compile Include="HttpListenerFactory.cs" />
```

Comment:
Nit: Was this moved up in order to alphabetize the list of files? If so, it's not in the right place ;)